docs/examples/real_world_example.rst

```diff
@@ -8,6 +8,12 @@ a web application might look like using the `Flask web framework
 <http://flask.pocoo.org/>`_ and GitHub as a provider. It should be easily
 transferrable to any web framework.
 
+.. note::
+  While the flow remains the same across msot providers, Github is special in that
```

Comment:
Can you bring this whole block up to the `.. note::` and then indent accordingly? That is:

```
.. note:: While the flow remains the same across most providers, GitHub
          is special in that...
```

Note also the spelling of `most`. =)